up-l1/zenoh.adoc

```diff
@@ -5,7 +5,7 @@
 The key words "*MUST*", "*MUST NOT*", "*REQUIRED*", "*SHALL*", "*SHALL NOT*", "*SHOULD*", "*SHOULD NOT*", "*RECOMMENDED*", "*MAY*", and "*OPTIONAL*" in this document are to be interpreted as described in https://www.rfc-editor.org/info/bcp14[IETF BCP14 (RFC2119 & RFC8174)]
 
 ----
-Copyright (c) 2023 General Motors GTO LLC
+Copyright (c) 2024 ZettaScale Technology
```

Comment:
I would suggest to use
```
SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
```
as suggested by https://www.eclipse.org/projects/handbook/#ip-copyright-headers and https://reuse.software/tutorial/#step-2 and then remove the redundant entry below the license header.

Reply:
Do you mean removing the line `SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation` and only use `Copyright (c) 2024 Contributors to the Eclipse Foundation` instead?